### ISSUE TITLE
Builds fail for me

### ISSUE THREAD

**Issue by jschmersal-cscc** [MarkABrown34/special-topics-labs-ci#1]
https://github.com/MarkABrown34/special-topics-labs-ci/blob/31c3408c6db5f7190c410ad81326c9cc89e25bc7/Jenkinsfile#L15

Hi Mark.  You're almost there, but when I try to build this I get:
```
INFO] [jenkins-event-spy] Generated /usr/share/jenkins/ref/jobs/build-pipeline-job/workspace@2@tmp/withMavena1c1822a/maven-spy-20191121-180304-8828951045896183371350.log
[ERROR] The goal you specified requires a project to execute but there is no POM in this directory (/usr/share/jenkins/ref/jobs/build-pipeline-job/workspace@2). Please verify you invoked Maven from the correct directory. -> [Help 1]
[ERROR] 
```

The problem is this extra `node` block that you have inserted.  Jenkins has a concept of worker nodes, and by having 2 `node` blocks it creates a separate workspace for the second worker to run this part that does the `mvn test` and reports the junit results.  However, that second workspace doesn't have a copy of the source code (only the first node actually downloaded the source).

TL;DR: Just remove this extra `node {` line and the corresponding close brace and it should work fine.

**Comment by MarkABrown34** [comment]
Thanks for reviewing this, Jeremy.  I've removed the offending node block and am submitting in Blackboard now.
